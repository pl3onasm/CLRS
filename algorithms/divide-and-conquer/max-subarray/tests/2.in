9 -1 2 9 4 -5 -4 36 -10 -15 20 9 -23 21 -35 -6 -27 6 3 10 1 10 -3 0 -5 5 -10 10 25 1 -36 4 -6 24 -10 6 4 -26 10 2