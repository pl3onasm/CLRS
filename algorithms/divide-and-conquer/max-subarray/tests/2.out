Maximum sum: 54
A subarray with the maximum sum:
  9, -1, 2, 9, 4, -5, -4, 36, -10, -15, 20, 9
